app/controllers/ExperimentController.scala

```diff
@@ -47,7 +47,7 @@ object ExperimentController extends Controller with RequestParser {
       jsObj <- requestKeyJson.asOpt[JsObject]
       (key, value) <- jsObj.fieldSet
     } {
-      body = body.replace(key, value.toString())
+      body = body.replace(key, value.toString().replace("\"",""))
```

Comment:
It may broken if type of `value` is not String.
ex) 
```javascript
// Request experiment with Object/Array type params
{
  "[[duration]]" : { "from": 0: "to": 10000 }
}
```
I Suggest this
```scala
value match {
  case JsString(s) => s
  case _ => value.toString
}
```